### PR TITLE
CB-2693 Enable Access Management tab for invalid license

### PR DESCRIPTION
**Pull request by tfokina**

### CODE REVIEW

File: webapp/packages/core-administration/src/AdministrationScreen/AdministrationScreenService.ts [hunk 1]
```diff
@@ -230,6 +230,10 @@ export class AdministrationScreenService {
       return true;
     }
 
+    if (fromState.params.item === 'license' && toState.params.item === 'users') {
```
Comment (devnaumov) [inline webapp/packages/core-administration/src/AdministrationScreen/AdministrationScreenService.ts]: I am not sure the CE product should know about the license route.

File: webapp/packages/plugin-authentication-administration/src/Administration/Users/UsersAdministrationService.ts [hunk 2]
```diff
@@ -39,6 +41,9 @@ export class UsersAdministrationService extends Bootstrap {
     this.administrationItemService.create({
       name: UsersAdministrationNavigationService.ItemName,
       order: 3,
+      isOnlyActive: configurationWizard => !configurationWizard
+      && !!this.serverConfigResource.licenseRequired
```
Comment (devnaumov) [inline webapp/packages/plugin-authentication-administration/src/Administration/Users/UsersAdministrationService.ts]: Do we need the double inversion of licenseRequired?

File: webapp/packages/core-administration/src/AdministrationScreen/AdministrationScreenService.ts (hunk 1 above)
Comment (Wroud) [reply webapp/packages/core-administration/src/AdministrationScreen/AdministrationScreenService.ts]: yep, we can't use plugins related code in core-* packages, we need to specify special behaviour via `IAdministrationItemOptions` in `administrationItemService.create`

**Review by Wroud — CHANGES_REQUESTED** [review]

File: webapp/packages/plugin-authentication-administration/src/Administration/Users/UsersAdministrationService.ts [hunk 3]
```diff
@@ -30,6 +31,7 @@ export class UsersAdministrationService extends Bootstrap {
     private readonly administrationItemService: AdministrationItemService,
     private readonly createUserService: CreateUserService,
     private readonly teamsResource: TeamsResource,
+    private readonly serverConfigResource: ServerConfigResource,
```
Comment (Wroud) [inline webapp/packages/plugin-authentication-administration/src/Administration/Users/UsersAdministrationService.ts]: not used

**Review by Wroud — CHANGES_REQUESTED** [review]

File: webapp/packages/plugin-authentication-administration/src/Administration/Users/UsersAdministrationService.ts (hunk 3 above)
Comment (tfokina) [reply webapp/packages/plugin-authentication-administration/src/Administration/Users/UsersAdministrationService.ts]: Removed unused property

**Review by Wroud — APPROVED** [review]

**Review by Wroud — APPROVED** [review]

**Pull request merged**